app/src/ui/branches/index.tsx

```diff
@@ -89,7 +95,7 @@ export class Branches extends React.Component<IBranchesProps, IBranchesState> {
         <span className="pull-request-tab">
           {__DARWIN__ ? 'Pull Requests' : 'Pull requests'}
 
-          <span className="count">{count}</span>
+          {countElement}
```

Comment:
`{this.props.pullRequests && <span className="count">{this.props.pullRequests.length}</span>}`? `false` is not rendered in React.

Reply:
It's probably just a taste thing, but that's always seemed a bridge too far for me.